### PR TITLE
added support for getDeferredApplink

### PR DESCRIPTION
**Pull request by jkassis**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,32 +1 @@
-*.DS_Store
```
Comment (jeduan) [inline .gitignore]: Any reason to remove all these entries?

File: docs/android/README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ $ cd myApp/
 $ cordova platform add android
 
 # Remember to replace APP_ID and APP_NAME variables
-$ cordova plugin add https://github.com/jeduan/cordova-plugin-facebook4 --save --variable APP_ID="123456789" --variable APP_NAME="myApplication"
+$ cordova plugin add https://github.com/jkassis/cordova-plugin-facebook4 --save --variable APP_ID="123456789" --variable APP_NAME="myApplication"
```
Comment (jeduan) [inline docs/android/README.md]: ????

**Comment by jeduan** [comment]
Please remove all extra files before we can move forward with this

**Comment by robertarnesson** [comment]
https://github.com/jeduan/cordova-plugin-facebook4/commit/f05ce3fdccd44023950e50fbd8c306b6a5349575

**Pull request closed without merging**